crates/ln-dlc-node/src/ldk_node_wallet.rs

```diff
@@ -103,12 +108,17 @@ where
         let locked_wallet = self.bdk_lock();
         let mut tx_builder = locked_wallet.build_tx();
 
-        let fee_rate = self.fee_rate_estimator.get(confirmation_target);
+        let fee_rate = self.fee_rate_estimator.estimate(confirmation_target);
         tx_builder
             .add_recipient(output_script, value_sats)
             .fee_rate(fee_rate)
             .enable_rbf();
 
+        let mut locked_out_points = self.locked_outpoints.lock();
```

Comment:
I guess we should implement the same logic at all places where we are building transactions. e.g. the split, glue transactions and commit transaction..

I wonder if it is an issue that we are managing utxos differently for within `ln_dlc_wallet`

https://github.com/get10101/10101/blob/776acb3bf142c30cdfd3a3dfec9fdda123e147a8/crates/ln-dlc-node/src/ln_dlc_wallet.rs#L269-L294